Jenkinsfile

```diff
@@ -157,6 +156,14 @@ pipeline {
 
                     def paths = ['stan', 'make', 'lib', 'test', 'runTests.py', 'runChecks.py', 'makefile', 'Jenkinsfile', '.clang-format'].join(" ")
                     skipRemainingStages = utils.verifyChanges(paths)
+
+                    if(!utils.verifyChanges("test/unit/math/opencl") || params.gpu_async){
```

Comment:
I apologize, I mislead you there. This should be 


`if(!utils.verifyChanges("stan/math/opencl") || !utils.verifyChanges("test/unit/math/opencl") || params.gpu_async){`

if verifyChanges returns false if there are changes in the folder (I am a bit confused with the negation there). Just double checking.

Reply:
Hey, fixed the paths. It is a bit misleading, first, it was used in this scenario:
```
def paths = ['stan', 'make', 'lib', 'test', 'runTests.py', 'runChecks.py', 'makefile', 'Jenkinsfile', '.clang-format'].join(" ")
skipRemainingStages = utils.verifyChanges(paths)
```
Might need to refactor all these one day.

Reply:
Ah, ok that makes sense. Good to go.